src/redux/slices/meetingSlice.js

```diff
@@ -1,19 +1,296 @@
+        headers: {
+          authorization: token,
+        },
+      });
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+// Route does not return the meeting that was deleted so the ex reducer won't be able to filter it out unless page is refreshed
+export const deleteMeeting = createAsyncThunk(
+  'meeting/deleteMeeting',
+  async ({ token, meetingId }, { rejectWithValue }) => {
+    try {
+      const res = axios.delete(API_URL + `/api/meeting/${meetingId}`, {
+        headers: {
+          authorization: token,
+        },
+      });
+      return { res, meetingId };
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const getMeetingMessages = createAsyncThunk(
+  'meeting/getMessages',
+  async ({ token, meetingId }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.get(
+        API_URL + `/api/meeting/${meetingId}/messages`,
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const addMessage = createAsyncThunk(
+  'meeting/addMessage',
+  async ({ token, meetingId, newMessage }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.post(
+        API_URL + `/api/meeting/${meetingId}/messages`,
+        newMessage,
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const addRating = createAsyncThunk(
+  'meeting/addRating',
+  async ({ token, meetingId, newRating }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.post(
+        API_URL + `/api/meeting/${meetingId}/rating`,
+        newRating,
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const upholdRating = createAsyncThunk(
+  'meeting/upholdRating',
+  async ({ token, ratingId, decision }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.put(
+        API_URL + `/api/rating/${ratingId}`,
+        { reportIsUpheld: decision },
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
 
 const meetingSlice = createSlice({
   name: 'meetings',
   initialState: {
     meetings: [],
     meeting: {},
+    isLoading: false,
+    error: '',
+    status: {},
   },
   reducers: {
     testMeeting: (state) => {
```

Comment:
I'll make a pass thru all of the slices and remove these now that we have "real" stuff to test against